benchmark/string_concat.yml

```diff
@@ -1,9 +1,9 @@
 prelude: |
   CHUNK = "a" * 64
-  BCHUNK = "a".b * 64
+  UCHUNK = "é" * 32
```

Comment:
Initially my benchmark used `"€"` which is actually 3 bytes, so my benchmark was slower than expected because we ended up reallocating the strings.

The benchmark now show that the 3 cases are all just as fast.